includes/class-wc-stripe.php

```diff
@@ -344,6 +344,11 @@ public function install() {
 				define( 'WC_STRIPE_INSTALLING', true );
 			}
 
+			// Mark as fresh install if no version is set.
+			if ( false === get_option( 'wc_stripe_version' ) && false === get_option( 'wc_stripe_fresh_install' ) ) {
+				update_option( 'wc_stripe_fresh_install', true );
+			}
```

Comment:
I think I would prefer a more specific name for the option, as it feels like this generic name could result in unexpected or unintended uses in future. Can we rename this to something explicitly mentioning Optimized Checkout? Given that is the only thing we are using it for, I think it's better to use a more specific option name.
```suggestion
			if ( false === get_option( 'wc_stripe_version' ) && false === get_option( 'wc_stripe_optimized_checkout_default_on' ) ) {
				update_option( 'wc_stripe_optimized_checkout_default_on', true );
			}
```

Reply:
I agree. My initial idea was to keep it generic in case we wanted to use this for Amazon Pay later as well, but since we are seeing if it can be enabled automatically on the Stripe side, we can keep it specific